build.gradle

```diff
@@ -4,13 +4,15 @@ buildscript {
     }
 
     dependencies {
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.1.3-2'
```

Comment:
I will in my next PR. Here is what I planned on doing next: https://github.com/KeepSafe/dexcount-gradle-plugin/pull/194.